.github/workflows/test.yml

```diff
@@ -183,12 +195,31 @@ jobs:
         platform: x86
         version: 12.2.0
     - uses: actions-rust-lang/setup-rust-toolchain@v1
+      if: ${{ ! contains(matrix.target, 'freebsd') }}
       with:
         toolchain: ${{matrix.toolchain}}
         target: ${{matrix.target}}
     - name: Test
+      if: ${{ ! contains(matrix.target, 'freebsd') }}
       run: cargo test --target ${{matrix.target}} --features ${{matrix.features}}
       env:
         CARGO_TARGET_x86_64-unknown-linux-musl: ${{matrix.rustflags}}
         CARGO_TARGET_i686-unknown-linux-musl: ${{matrix.rustflags}}
         CARGO_TARGET_aarch64-unknown-linux-musl: ${{matrix.rustflags}}
+    - name: Test FreeBSD
+      if: ${{ contains(matrix.target, 'freebsd') }}
+      uses: vmactions/freebsd-vm@v1
+      with:
+        release: "13.2"
+        usesh: true
+        mem: 8192
+        copyback: false
+        prepare: |
+          pkg install -y curl ca_root_nss
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain ${{matrix.toolchain}}
+        run: |
+          . "${HOME}/.cargo/env"
+          # Try to avoid https://github.com/rustls/rustls-native-certs/issues/28
+          export SSL_CERT_FILE=/usr/local/share/certs/ca-root-nss.crt
```

Comment:
I wonder if this would be unnecessary if `native-tls` was used on FreeBSD as well as macOS? IIRC, I included this to work around issues with `rustls-tls-native-roots`. I don't need to set that when building locally though.

https://github.com/JuliaLang/juliaup/blob/4f6196e069b6518785b1443c59c8f60dd51112be/Cargo.toml#L66-L70

Reply:
@davidanthoff, do you recall why `native-tls` is used on macOS and `rustls-tls-native-roots` is used elsewhere (except Windows)?

Reply:
I don't... Maybe `native-tls` didn't work reliably on all Linux versions?

In my mind, `native-tls` is better if it works, as that (I think) uses the native OS trust verification, which can be more thorough than just using the OS list of certificates. So I'd say that if `native-tls` works on FreeBSD we should go with that.

Reply:
Well, it looks like it works.